api/docs/http_file_metadata.md

```diff
@@ -12,7 +12,7 @@ the following three types:
 * symbolic link
 
 Note that an `:environment` must be specified in the endpoint, but is actually ignored since the puppet file server
-is not environment-specific.  (In fact, the specified `:environment` does even need to be valid.)
+is not environment-specific.
```

Comment:
So it sounds like I should probably just remove this paragraph since it's not really true at all, or I could try to say something about where `:environment` is ignored and where it isn't? But trying to explain that seems like it might be more confusing than helpful....